src/index.html

```diff
@@ -1,14 +1,38 @@
 <!doctype html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="logo"></a>
```

Comment:
apply code styling recommendations from checklist all over the page
```suggestion
      <a href="#home" class="logo">
        <img src="images/logo.png" alt="logo">
      </a>
```

Reply:
done